test/intern/mixins/ColumnSet.js

```diff
@@ -3,16 +3,50 @@ define([
 	'intern/chai!assert',
 	'dojo/_base/declare',
 	'dojo/aspect',
+	'dojo/Deferred',
+	'dojo/on',
 	'dojo/query',
 	'dgrid/test/data/createSyncStore',
+	'dgrid/test/data/createHierarchicalStore',
 	'dgrid/OnDemandGrid',
 	'dgrid/ColumnSet',
 	'dgrid/Keyboard',
+	'dgrid/Tree',
+	'dgrid/util/misc',
 	'../addCss'
-], function (test, assert, declare, aspect, query, createSyncStore, OnDemandGrid, ColumnSet, Keyboard) {
+], function (test, assert, declare, aspect, Deferred, on, query, createSyncStore, createHierarchicalStore,
+	OnDemandGrid, ColumnSet, Keyboard, Tree, miscUtil) {
+
 	var grid;
 	var handles;
 
+	var expand = function (id) {
+		var dfd = new Deferred();
+		grid.expand(id);
+		dfd.resolve();
+		return dfd.promise;
+	};
+
+	var collapse = function (id) {
+		var dfd = new Deferred();
+		grid.expand(id, false);
+		dfd.resolve();
+		return dfd.promise;
+	};
+
+	function scrollDistance(node, direction, distance) {
```

Comment:
Since direction only ever has one of two values, make it a boolean, e.g. `isHorizontal`.